core/libraries/batch/JobHandlers/RegistrationsReport.php

```diff
@@ -662,19 +492,20 @@ public function get_csv_data_for($event_id, $offset, $limit, $question_labels, $
     /**
      * Counts total unit to process
      *
-     * @deprecated since 4.9.19
-     * @param int|array $event_id
+     * @param array $query_params
      * @return int
+     * @throws EE_Error
      */
-    public function count_units_to_process($event_id)
+    public function count_units_to_process($query_params)
     {
-        // use the legacy filter
-        if ($event_id) {
-            $query_params[0]['EVT_ID'] = $event_id;
-        } else {
-            $query_params['force_join'][] = 'Event';
-        }
-        return EEM_Registration::instance()->count($query_params);
+        return EEM_Registration::instance()->count(
+            array_diff_key(
+                $query_params,
+                array_flip(
+                    ['limit']
+                )
+            )
+        );
     }
```

Comment:
hmmm... this is a big change... 🤔 
was this method not being used before? IF you are 1000% certain this was not being used anywhere else (ie: search for the method name directly as opposed to relying on your IDE to know) then this is great!

Reply:
Yes, I already confirmed it by searching the function name on whole `plugins` directory.

Reply:
awesome thnx 👍🏻